examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_2.ipynb

```diff
@@ -0,0 +1,339 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../../../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 2: Speed improvements\n",
+    "\n",
+    "**GOAL**: Learn about common functionality that Modin speeds up by using all of your machine's cores."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for Exercise: `read_csv` speedups\n",
+    "\n",
+    "The most commonly used data ingestion method used in pandas is CSV files (link to pandas survey). This concept is designed to give an idea of the kinds of speedups possible, even on a non-distributed filesystem. Modin also supports other file formats for parallel and distributed reads, which can be found in the documentation.\n",
+    "\n",
+    "We will import both Modin and pandas so that the speedups are evident.\n",
+    "\n",
+    "**Note: Rerunning the `read_csv` cells many times may result in degraded performance, depending on the memory of the machine**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import modin.pandas as pd\n",
+    "import pandas\n",
+    "import time\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put(\"omnisci\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dataset: 2015 NYC taxi trip data\n",
+    "\n",
+    "Link to raw dataset: https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page\n",
+    "\n",
+    "We will be using a version of this data already in S3, originally posted in this blog post: https://matthewrocklin.com/blog/work/2017/01/12/dask-dataframes\n",
+    "\n",
+    "**Size: ~2GB**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = \"s3://dask-data/nyc-taxi/2015/yellow_tripdata_2015-01.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Optional:** Note that the dataset takes a while to download. To speed things up a bit, if you prefer to download this file once locally, you can run the following code in the notebook:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# [Optional] Download data locally. This may take a few minutes to download.\n",
+    "# import urllib.request\n",
+    "# url_path = \"https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv\"\n",
```

Comment:
```suggestion
    "# url_path = \"https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2015-01.csv\"\n",
```
aligning path with the cell above (please verify that it's valid and works properly)

Reply:
@dchigarev I changed this